server/middleware/analytics-log-events.coffee

```diff
@@ -0,0 +1,14 @@
+wrap = require 'co-express'
+AnalyticsLogEvent = require '../models/AnalyticsLogEvent'
+
+post = wrap (req, res) ->
+  # Converts strings to string IDs where possible, and logs the event
+  user = req.user?._id
+  { event, properties } = req.body
+  eventRes = AnalyticsLogEvent.logEvent user, event, properties
```

Comment:
Does the `doc.save()` in logEvent happen synchronously and thus able to return an error, or no? Should it return a error if it fails? I'm guessing not since we wouldn't want the supermodel complaining about a failed network request.

Reply:
It may be good to add a spec for the right behavior (returning success or error when sending an invalid event)